harshita/level1/findvalue.cpp

```diff
@@ -0,0 +1,18 @@
+#include<iostream>
+using namespace std;
+
+
+int main()
+{
+	int x = 2,y = 5,z = 0;
+	cout << (x==2);
+	cout <<  (x != 5);
+    cout << (x != 5 && y >= 5);
+    cout << (z != 0 || x == 2);
+    cout << (!(y < 10));
+	    
```

Comment:
Don't add unnecessary lines and don't forget to add `return 0;` in main function.